docs/self-host.docker

```diff
@@ -0,0 +1,16 @@
+FROM vladikoff/fxa-slim-image:latest
+
+RUN adduser --disabled-password --gecos '' fxa && adduser fxa sudo && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+COPY . /home/fxa/fxa-profile-server
+WORKDIR /home/fxa/fxa-profile-server
+RUN chown -R fxa .
+USER fxa
+
+RUN npm install --production \
+  && npm cache clear
+
+CMD npm start
+
+# Expose ports
+EXPOSE 1111
```

Comment:
I ran these through the Dockerfile linter at http://hadolint.lukasmartinelli.ch/ and it only came back with the following two warnings:

> 1. [**DL3007**](https://github.com/lukasmartinelli/hadolint/wiki/DL3007) Using latest is prone to errors if the image will ever update. Pin the version explicitely to a release tag.
> 2. [**DL4000**](https://github.com/lukasmartinelli/hadolint/wiki/DL4000) Specify a maintainer of the Dockerfile

I imagine we can safely ignore (1) since it's managed by Vlad, but not sure if we wanted to add a `MAINTAINER` field.